unblob/processing.py

```diff
@@ -52,14 +53,15 @@ def process_file(
         depth=0,
     )
 
-    processor = Processor(
-        extract_root,
-        max_depth,
-        entropy_depth,
-        entropy_plot,
-        handlers,
-        keep_extracted_chunks,
+    parameters = Parameters(
```

Comment:
I'd consider lifting this `(Processing)Parameters` config object upper one level, so that `process_file` expects this as an argument. You can also add the defaults values to it.

Reply:
This is a valid improvement, but maybe better done in another PR.

Reply:
As discussed, it would also be better named as `ExtractConfig`, it will be in a follow-up PR